.gitignore

```diff
@@ -136,12 +135,6 @@ csx/
 # Windows Store app package directory
 AppPackages/
 
-# Visual Studio cache files
-# files ending in .cache can be ignored
-*.[Cc]ache
```

Comment:
Why do we care about .cache files now?

Reply:
We don't, particularly. As I tried to mention in the commit message, some of our test assets were excluded by the pattern here and the easiest thing to do was drop it (since all of the actually generated cache files are excluded by the `artifacts/` pattern anyyway.